drivers/hmis/app/controllers/hmis/graphql_controller.rb

```diff
@@ -100,5 +116,23 @@ def handle_graphql_exception(err)
         data: {},
       }
     end
+
+    def graphql_activity_log(gql_param)
+      {
+        user_id: current_hmis_user.id, # FIXME: true_user if masquerading
```

Comment:
Can this be fixed now, or is it dependent on unmerged code?

Reply:
Maybe a `TodoOrDie`? with a near future end-date?  Seems like something we don't want to forget.

Reply:
Good point, I can add a TodoOrDie; it depends on https://github.com/greenriver/hmis-warehouse/pull/3672